cypress/support/index.js

```diff
@@ -1 +0,0 @@
-import 'cypress-vue-unit-test/support'
```

Comment:
this delete looks suspicious

Reply:
It was breaking the e2e tests. I was also suspicious. @bahmutov what's this line doing for us? I don't see a support file.

Reply:
we need to load `suport` file which creates a placeholder element in the iframe, which we do in
we point at that support file here https://github.com/bahmutov/cypress-vue-unit-test/pull/247/files#diff-73ffe0d8ebfaf5bdc6b0546e41f576c0R11 using an alias to avoid coding `../../support` https://github.com/bahmutov/cypress-vue-unit-test/pull/247/files#diff-6d4bb79610683d9749d2cbb62e708d88

If this breaks, land this PR after #287 and we can check out the merged result to see if it works correctly

Reply:
Seemingly no issues after merging that PR... will continue merging and then validate again